DatadogCore/Tests/Datadog/RUM/Integrations/CrashReportReceiverTests.swift

```diff
@@ -551,6 +552,174 @@ class CrashReportReceiverTests: XCTestCase {
+
+        // Given
+        let crashDate: Date = .mockDecember15th2019At10AMUTC()
+        let crashReport: DDCrashReport = .mockWith(
+            date: crashDate,
+            type: "SIG_CODE (SIG_NAME)",
+            message: "Signal details",
+            stack: """
+            0: stack-trace line 0
+            1: stack-trace line 1
+            2: stack-trace line 2
+            """,
+            threads: [
+                .init(name: "Thread 0", stack: "thread 0 stack", crashed: true, state: nil),
+                .init(name: "Thread 1", stack: "thread 1 stack", crashed: false, state: nil),
+                .init(name: "Thread 2", stack: "thread 2 stack", crashed: false, state: nil),
+            ],
+            binaryImages: [
+                .init(libraryName: "library1", uuid: "uuid1", architecture: "arch", isSystemLibrary: true, loadAddress: "0xLoad1", maxAddress: "0xMax1"),
+                .init(libraryName: "library2", uuid: "uuid2", architecture: "arch", isSystemLibrary: true, loadAddress: "0xLoad2", maxAddress: "0xMax2"),
+                .init(libraryName: "library3", uuid: "uuid3", architecture: "arch", isSystemLibrary: false, loadAddress: "0xLoad3", maxAddress: "0xMax3"),
+            ],
+            meta: .init(
+                incidentIdentifier: "incident-identifier",
+                process: "process [1]",
+                parentProcess: "parent-process [0]",
+                path: "process/path",
+                codeType: "arch",
+                exceptionType: "EXCEPTION_TYPE",
+                exceptionCodes: "EXCEPTION_CODES"
+            )
+        )
+
+        let dateCorrectionOffset: TimeInterval = .mockRandom(min: 1, max: 5)
+        let crashContext: CrashContext = .mockWith(
+            serverTimeOffset: dateCorrectionOffset,
+            trackingConsent: .granted,
+            lastRUMViewEvent: AnyCodable(lastRUMViewEvent) // means there was a RUM session and it was sampled
+        )
+
+        let receiver: CrashReportReceiver = .mockWith(
+            dateProvider: RelativeDateProvider(
+                using: crashDate.addingTimeInterval(
+                    .mockRandom(min: 10, max: 2 * CrashReportReceiver.Constants.viewEventAvailabilityThreshold) // simulate restarting app from 10s to 8h later
+                )
+            ),
+            sessionSampler: Bool.random() ? .mockKeepAll() : .mockRejectAll(), // no matter sampling (as previous session was sampled)
+            trackBackgroundEvents: .mockRandom(), // no matter BET
+            eventsMapper: .mockWith(
+                errorEventMapper: { _ in
+                    return nil
+                }
+            )
+        )
+
+        // When
+        XCTAssertTrue(
+            receiver.receive(message: .baggage(
+                key: MessageBusSender.MessageKeys.crash,
+                value: MessageBusSender.Crash(report: crashReport, context: crashContext)
+            ), from: core)
```

Comment:
This doesn't compile. With my latest changes on `develop`, we should now test against `FeatureScopeMock` not the `core`. See surrounding tests in this file, I updated them this morning.

Reply:
Ugh, sorry, pulled and rebased and didn't see any conflicts so assumed it was good. Fixed now.